src/components/map/rent-controls.tsx

```diff
@@ -84,6 +86,19 @@ const RentControls: React.FC<RentControlsProps> = ({
               !isOpenedStationBooked ? `(${bookedStation!.name})` : ''
             }`}
       </button>
+      {booking && (
+        <button
+          className="btn outline book"
+          onClick={onRefreshBooking}
+        >
+          {isOpenedStationBooked
+            ? `${map.BOOKING.REFRESH} (${
+              (new Date(Date.parse(booking.expiryDateTime) - Date.now())).getUTCMinutes()
+            } ${map.BOOKING.MINUTES_REMAINING})`
+            : map.BOOKING.SWITCH
+          }
+        </button>
+      )}
```

Comment:
So this button just appears below the existing one? Okay for me per-se, but I think this will cause plopping in the UI (which is bad). Could we instead show the button next to the exising one (so on the same line, hint: flexbox with `flex-grow: 1`)? I'm afraid it might not fit on mobile, but would be great if it did.

Reply:
I already gave the plopping UI a thought and I have to say the only real viable option would be to disable it through `disabled`. During the canceling and rebooking the UI will plop nevertheless if we don't (there is a moment in between where `booking` is falsy). Or the loading dots will need to be reintroduced (don't know when they were "removed") - in the past they showed up e.g. when canceling a booking from another station.

The side-by-side layout was my very first idea, but unfortunately, especially with the additional information (remaining time) and the german translation being longer, I don't see it working out on mobile.

Reply:
> the only real viable option would be to disable it through `disabled`

15e3864af1fabacf95cd5c5601b3b2848330b475 demonstrates this option.

Reply:
> don't know when they were "removed"

I removed them during the refactoring of the popup because they weren't necessary anymore and because the process of reloading the data was cleaned up. When you want to reload some part of the data, just call the corresponding fetch-function, the shenanigans with "pseudo-opening" and -closing the popup is gone.

> The side-by-side layout was my very first idea, but unfortunately, especially with the additional information (remaining time) and the already long labeling, I don't see it working out on mobile.

That is a pity! Could we perhaps show the remaining time somewhere else (maybe in the bike icon or the header?) than on the button and shorten the button labels a bit? I'd really prefer the UI not to plop.

Reply:
![image](https://user-images.githubusercontent.com/23734747/54878268-9d876580-4e2a-11e9-963a-1d681865d40c.png)
I tried adding a paragraph to free the information from the button labels, but I think it needs to be a whole sentence... It was fine until I hit a station name containing sponsoring...
![image](https://user-images.githubusercontent.com/23734747/54878248-4a151780-4e2a-11e9-8984-965629ef97b4.png)

Reply:
Ugh, yeah, let's not do that for the moment.